config.js

```diff
@@ -1755,9 +1755,7 @@ var config = {
     //     // How long the GIF should be displayed on the tile (in milliseconds).
     //     tileTime: 5000,
     //     // Limit results by rating: g, pg, pg-13, r. Default value: g.
-    //     rating: 'pg',
-    //     // The proxy server url for giphy requests in the web app.
-    //     proxyUrl: 'https://giphy-proxy.example.com',
+    //     rating: 'pg'
```

Comment:
Nit: leave the trailing comma, so adding new options doesn't result in a syntax error.

Reply:
Done!